README.rst

```diff
@@ -128,31 +128,19 @@ Browser Extensions
 ^^^^^^^^^^^^^^^^^^
 Run the following command at the prompt to build the Chrome extension::
 
-    $ ./bin/hypothesis extension development.ini chrome http://localhost:5000
+    $ hypothesis-buildext development.ini chrome
```

Comment:
There's [a default](https://github.com/hypothesis/h/pull/1993/files#diff-13b93f8707dd71549f3e7f8903f34199R259) of `http://localhost:5000/`.